src/extension.ts

```diff
@@ -224,19 +221,19 @@ export async function activateInternal(
         registerCommand('appService.StartRemoteDebug', startRemoteDebug);
         registerCommand('appService.StartSsh', startSsh);
 
-        registerCommand('appService.showFile', async (_actionContext: IActionContext, node: FileTreeItem) => { await showFile(node, fileEditor); }, 500);
+        registerCommand('appService.openFile', async (_actionContext: IActionContext, node: FileTreeItem) => { await showFile(node, fileEditor); }, 500);
         registerCommand('appService.ScaleUp', async (_actionContext: IActionContext, node: DeploymentSlotsNATreeItem | ScaleUpTreeItem) => {
             await openInPortal(node.root, node.scaleUpId);
         });
 
         registerEvent('appService.fileEditor.onDidSaveTextDocument', vscode.workspace.onDidSaveTextDocument, async (actionContext: IActionContext, doc: vscode.TextDocument) => { await fileEditor.onDidSaveTextDocument(actionContext, context.globalState, doc); });
-        registerCommand('appService.EnableFileLogging', async (actionContext: IActionContext, node?: SiteTreeItem | FolderTreeItem) => {
+        registerCommand('appService.EnableFileLogging', async (actionContext: IActionContext, node?: SiteTreeItem | LogFilesTreeItem) => {
             if (!node) {
                 node = <SiteTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue, actionContext);
             }
 
-            if (node instanceof FolderTreeItem) {
-                // If the entry point was the Files/Log Files node, pass the parent as that's where the logic lives
+            if (node instanceof LogFilesTreeItem) {
```

Comment:
Is this not the case anymore?

Reply:
Can you expand on what you mean by "this"? If you're talking about how I deleted "Files/" from the comment below, I don't think we ever had "Enable File Logging" on that node - only the "Logs" node. See https://github.com/microsoft/vscode-azureappservice/blob/master/package.json#L546

Reply:
Yeah, that's what I meant.  That makes sense-- FolderTreeItem used to be both the Files and Log Files, I believe.